patroni/dcs/__init__.py

```diff
@@ -1698,6 +1698,16 @@ def reset_cluster(self) -> None:
             self._cluster = None
             self._cluster_valid_till = 0
 
+    def ensure_cluster_access(self, sleep_time: int = 5) -> Optional[Cluster]:
```

Comment:
```suggestion
    def ensure_dcs_access(self, sleep_time: int = 5) -> Cluster:
```

Reply:
And why this method is in AbstractDCS and not in Patroni class?

Reply:
I originally thought that all relevant access methods should be provided by the DCS class, but it seems more appropriate for the Patroni class which is the only user to provide it.